history.md

```diff
@@ -0,0 +1,133 @@
+
+During this discussion,
+it was decided that the dojo wonld not be owned by a company.
+
+The dojo will need a logistical support (room, projector, chairs …),
+and [Epiconcept](https://www.epiconcept.fr/en/) was the first to provide it.
+
+## The beginnings
+
+On [<time datetime="2004-12-13">2004-12-13</time>](https://web.archive.org/web/20081118133051/http://xp-france.net/cgi-bin/wiki.pl?DojoDeveloppement/DojoZero),
+the first dojo took place,
+the participants practised in groups of two.
+
+On [<time datetime="2005-01-10">2005-01-10</time>](https://web.archive.org/web/20050110063104/http://xp-france.net/cgi-bin/wiki.pl?DojoDeveloppement),
+like all the following dojos,
+the second dojo began with a retro,
+the first dojo was horrible according to everyone,
+they decided to practice together in a united group.
+
+Since 2004, a new dojo has been held almost every week.
+
+## Archives
+
+The group archived its work on [the xp-france.net wiki](https://web.archive.org/web/20091209141145/http://xp-france.net/cgi-bin/wiki.pl?DojoDeveloppement),
+which has since disappeared ;
+they moved to [the agile-france.org wiki](https://web.archive.org/web/20100424015604/http://wiki.agile-france.org/cgi-bin/wiki.pl?DojoDeveloppement).
+
+They started archiving their sessions in a git repository hosted on [Github](https://github.com/dojo-developpement-paris/dojo-developpement-paris.github.io),
+since the session of [<time datetime="2015-01-15">2015-01-15</time>](https://github.com/dojo-developpement-paris/dojo-developpement-paris.github.io/tree/main/2015/01/05).
+
+They started using [Meetup](https://www.meetup.com/fr-FR/dojo-developpement-paris/),
+since the session of [<time datetime="2015-01-19">2015-01-19</time>](https://www.meetup.com/fr-FR/dojo-developpement-paris/events/219806503/).
+
+## Lockdown
+
+The last dojo before the lockdown was [<time datetime="2020-03-12">2020-03-12</time>](https://www.meetup.com/dojo-developpement-paris/events/269323464/).
+
+The lockdown paused the dojo.
+
+There was an attempt to resume online
+from [<time datetime="2021-01-28">2021-01-28</time>](https://www.meetup.com/dojo-developpement-paris/events/275980672/) to [<time datetime="2021-06-17">2021-06-17</time>](https://www.meetup.com/dojo-developpement-paris/events/278752692/).
+
+The dojo has been back in person since [<time datetime="2022-05-13">2022-05-13</time>](https://www.meetup.com/dojo-developpement-paris/events/285807516/).
+
+## Mob Programming
+
+As usual, they practised as a whole group.
+
+On one occasion, [Woody Zuill](https://woodyzuill.com/) took part
+and said something like :
+
+> You should use this organisation in your company !
+
+Since he worked at Hunter Industries,
+developed this type of organisation,
+and then popularised it as [Mob Programming](https://en.wikipedia.org/wiki/Team_programming#Mob_programming),
+on <time datetime="2012-09-06">2012-09-06</time>, he wrote [his first post about it](https://mobprogramming.org/hello-world/).
```

Comment:
```suggestion
Since he worked at Hunter Industries and,
inspired by this type of organisation,
created and popularised [Mob Programming](https://en.wikipedia.org/wiki/Team_programming#Mob_programming),
on <time datetime="2012-09-06">2012-09-06</time>, he wrote [his first post about it](https://mobprogramming.org/hello-world/).
```

Reply:
j'ai retiré la section en attendant d'etre certain de savoir comment ça s'est passé